server/index.js

```diff
@@ -33,8 +33,14 @@ app.put('/products/:productId', (req, res) => {
   });
 });
 
+<<<<<<< HEAD
+app.delete('/products', (req, res) => {
+  console.log('req', req);
+  models.deleteProduct(req.query.productId, (err, results) => {
+=======
 app.delete('/products/:productId', (req, res) => {
   models.deleteProduct(req.params.productId, (err, results) => {
+>>>>>>> 033cd84b675e660067a29f21d72fed179cea34f0
```

Comment:
Looks like there are some merge conflicts if you haven't gotten to them yet!

Reply:
Thanks, fixed them if you get a chance to review again.